java/daikon/dcomp/DCInstrument.java

```diff
@@ -1811,6 +1842,51 @@ void add_exit(MethodGen mg, MethodInfo mi, int method_info_index) {
     }
   }
 
+  /**
+   * Search interfaces for implementation of target method.
+   *
+   * @param targetClass JavaClass to search
+   * @param method_name target method to search for
+   * @param arg_types the target method's argument types
+   * @return name of class containing target method or null if not found
+   */
+  @ClassGetName String search_interfaces(JavaClass targetClass, String method_name, Type[] arg_types) {
+
+    // debug code
+    // System.out.println("search_interfaces of: " + targetClass.getClassName());
+    for (String interfac_temp : targetClass.getInterfaceNames()) {
```

Comment:
Try puting `@SuppressWarnings("signature:assignment")
      @ClassGetName` after the open parenthesis.

Reply:
Could not get this to work as then conflict with @ClassGetName interfac_temp and @SignatureUnknown getInterfaceNames().